### ISSUE TITLE
Instructional design

### ISSUE THREAD

**Issue by Protohedgehog** [OpenScienceMOOC/Main#10]
How to customise the MOOC for different learning groups at an institutional level. For example, creating Beginner, Intermediate, and Expert levels.

**Comment by jcolomb** [comment]
Maybe think of it inside a module too. What about module 5: task 1 and 2 for beginners, task 3 for intermediates ?
Will it need to be then split into 2 Eliadamy courses, with 2 certificates ?

**Comment by Protohedgehog** [comment]
I don't think that's too good an idea, to split modules like that. It becomes much more difficult to complete them then, and to provide certificates.

**Comment by jcolomb** [comment]
I disagree, it is easier to complete, you just need 3 different certificates for each module. But maybe there is something I am missing here... Maybe we need a more comprehensive discussion, see #25
